geoh5py/objects/surveys/direct_current.py

```diff
@@ -213,6 +272,64 @@ def metadata(self, values):
         self._metadata = values
         self.workspace.update_attribute(self, "metadata")
 
+    @property
+    @abstractmethod
+    def potential_electrodes(self):
+        """
+        The associated potential_electrodes (receivers)
+        """
+
+
+class PotentialElectrode(BaseElectrode):
```

Comment:
*question (out-of-scope):** from another PR? out of scope for this PR

Reply:
Allowed me to reduce code duplication for the copy by creating a sub-class. It's definitely a bit of a creep, but overall less lines to test.